packages/atomic/src/pages/index.html

```diff
@@ -288,6 +288,17 @@
       <div class="results">
         <atomic-did-you-mean></atomic-did-you-mean>
         <atomic-result-list fields-to-include="snrating,sncost">
+          <atomic-result-template image="small" must-match-sourcetype="YouTube">
```

Comment:
More of an open question really, should the `image` attribute be placed on the `template`, or on the `atomic-result-section-visual`?

Benefits of scoping could be:
- If the template is redesigned with different sections, it avoids having left-over attributes on `atomic-result-template`.
- If someone places an image in a section other than `atomic-result-section-visual`, would `image` have no effect? If so, I could see someone reporting this as a bug.
- Multiple images/icons with different sizes in a single template?

Reply:
> If someone places an image in a section other than atomic-result-section-visual, would image have no effect? If so, I could see someone reporting this as a bug.

Good point, all `image` does is resize `atomic-result-section-visual`.

> Multiple images/icons with different sizes in a single template?

There can't be multiple visual sections in a result.